StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/AuthFlowHelpers.swift

```diff
@@ -36,10 +36,17 @@ final class AuthFlowHelpers {
         url: URL,
         pane: FinancialConnectionsSessionManifest.NextPane,
         analyticsClient: FinancialConnectionsAnalyticsClient,
-        handleStripeScheme: (_ urlHost: String?) -> Void
+        handleURL: (_ urlHost: String?, _ nextPaneOrDrawerOnSecondaryCta: String?) -> Void
     ) {
-        if let urlParameters = URLComponents(url: url, resolvingAgainstBaseURL: true),
-            let eventName = urlParameters.queryItems?.first(where: { $0.name == "eventName" })?.value
+        let internalLinkToPaneId: [String: String] = [
+            "manual-entry": "manual_entry"
+        ]
```

Comment:
this mapping is a bit unfortunate but this is what [stripe.js decided to do](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/4c7be206/src/linkedAccounts/shared-ui/v3/typography/serverTextParser.tsx#L38-L40) so just following the lead